tests/test_chandra.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from astropy.coordinates import ICRS
+from astropy.units import Quantity
+
+from daxa.mission import Chandra
+
+defaults = Chandra()
+
+class TestChandra(unittest.TestCase):
+    def test_valid_inst_selection(self):
+        mission = Chandra(insts=['ACIS-I', 'ACIS-S'])
+        self.assertEqual(mission.chosen_instruments, ['ACIS-I', 'ACIS-S'])
+
+    def test_not_implemented_insts(self):
```

Comment:
I removed the not implemented errors for the Chandra gratings (though we don't really 'support' them yet, I didn't see why they shouldn't be allowed to be chosen.